com.unity.render-pipelines.high-definition/Runtime/ShaderLibrary/UpsampleTransparent.shader

```diff
@@ -55,7 +55,7 @@ Shader "Hidden/HDRP/UpsampleTransparent"
             float4 lowResDepths = GATHER_RED_TEXTURE2D_X(_LowResDepthTexture, s_linear_clamp_sampler, ClampAndScaleUVForBilinear(uv, halfResTexelSize));
 
             // Gather UVs
-            float2 topLeftUV = uv - 0.5f * halfResTexelSize;
+            float2 topLeftUV = uv - halfResTexelSize;
```

Comment:
This way we have a mismatch with the UVs used for gather I think. They are here so that they match the gather result.